src/flows/Termination/hooks.ts

```diff
@@ -215,15 +215,15 @@ export const useTermination = ({
      */
     isLoading: isLoadingTermination,
     /**
-     * Loading state indicating if a contract amendment mutation is in progress
+     * Loading state indicating if the termination mutation is in progress
      */
     isSubmitting: createTermination.isPending,
     /**
      * Initial form values
      */
     initialValues: initialValues,
     /**
-     * Function to validate form values against the contract amendment schema
+     * Function to validate form values against the termination schema
```

Comment:
Should be done in a separate PR, since this one is targeting `onboarding-flow` branch